src/main/java/servlets/RegisterServlet.java

```diff
@@ -23,8 +23,14 @@ protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws S
         String email = req.getParameter("email");
         String password = req.getParameter("password");
 
+
         if (ObjectUtils.allNotNull(firstName, lastName, email, password)) {
-            userService.create(new User(firstName, lastName, email, UserRole.USER.toString(), password));
+            userService.create(new User.Builder()
+                    .withFirstName(firstName)
+                    .withLastName(lastName)
+                    .withEmail(email)
+                    .withPassword(password)
+                    .build());
```

Comment:
I don't see code that will make user default role to USER. It could be implemented in setter.